src/aura/Grid/GridHelper.js

```diff
@@ -0,0 +1,93 @@
+({
+	read: function (component, event) {
+		var status = component.get("v.status");
+		var typeTask = component.get("v.typeTask");
+		var idContact = component.get("v.idContact");
+		// Create the action
+		
+		var action = component.get("c.getItems");
+		action.setParams({
+			"id": idContact,
+			"typeTask": typeTask,
+			"status": status,			
+		});
+		// Add callback behavior for when response is received
+		action.setCallback(this, function (response) {
+			var state = response.getState();
+			if (state === "SUCCESS") {
+				component.set("v.tasks", response.getReturnValue());
+			}
+			else {
+				console.log("Failed with state: " + state);
+			}
+		});
+
+		// Send action off to be executed
+		$A.enqueueAction(action);
+
+	},
+
+	updatedStatus: function (component, event, status) {
+		// Update
+		var habit = component.get("v.buttonHabit");
+		var id = event.getSource().get("v.value");
+		var action = component.get("c.updateHabit");
+		action.setParams({
+			"id": id,
+			"status": status,
+			"habit": habit
+		});
+		// Add callback behavior for when response is received
+		action.setCallback(this, function (response) {
+			var state = response.getState();
+			if (state === "SUCCESS") {
+				component.set("v.before", response.getReturnValue());
```

Comment:
Why do you need to load ALL tasks again after a single task update? I would recommend you to re-shape this code and make your UI react to changes in data rather than force the reaction through a full server-side load.